UnityProject/Assets/Scripts/SpawnPointList.cs

```diff
@@ -63,15 +66,38 @@ public SpawnPointListItem AddItem(string text)
       return listItem;
     }
 
+    public IEnumerator MoveContentToPosition(GameObject selectedObj)
+    {
+      var listContentTransform = ContentTransform.GetComponent<RectTransform>();
+      var targetPosition = new Vector2(listContentTransform.anchoredPosition.x, GetTargetContentPosition(selectedObj));
+      var startPosition = new Vector2(listContentTransform.anchoredPosition.x, listContentTransform.anchoredPosition.y);
+      var time = 0f;
+
+      while (time < _movementDuration)
+      {
+        ContentTransform.anchoredPosition = Vector2.Lerp(startPosition, targetPosition, time / _movementDuration);
+        time += Time.deltaTime;

```

Comment:
might be better to cache the startTime and just subtract it from currentTime? The while condition would also need to change, but it's a little cleaner imo